packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -264,57 +278,17 @@ contract OPStackManager is ISemver, Initializable {
         upgradeAndCall(output.opChainProxyAdmin, address(output.optimismMintableERC20FactoryProxy), impl.logic, data);
 
         impl = getLatestImplementation("L1CrossDomainMessenger");
-        // TODO add this check back in
```

Comment:
Do we not want to have this check in at some point? Is the reason it's being removed because we're wiring everything together inside this function and there's no need to sanity check?